openedx/core/djangoapps/ace_common/apps.py

```diff
@@ -19,7 +19,7 @@ class AceCommonConfig(AppConfig):
             ProjectType.CMS: {
                 SettingsType.AWS: {PluginSettings.RELATIVE_PATH: u'settings.aws'},
                 SettingsType.COMMON: {PluginSettings.RELATIVE_PATH: u'settings.common'},
-                SettingsType.DEVSTACK: {PluginSettings.RELATIVE_PATH: u'settings.devstack'},
+                SettingsType.DEVSTACK: {PluginSettings.RELATIVE_PATH: u'settings.common'},
```

Comment:
Can we simply remove the configuration type? @andrey-canon

Reply:
from line 19 to 22 ??? yes, this is only code from the backport but I think it is not necessary @Squirrel18

Reply:
No, I mean, can we remove the DEVSTACK setting type, we are not using it. @andrey-canon

Reply:
the line defines the plugin setting for devstack, but the settings are been defined  with a different file, it doesn't mean that the settings are not been using @Squirrel18

Reply:
If the DEVSTACK settings are not defined, the platform will use a COMMON settings, right? @andrey-canon

Reply:
I don't think so, but the true is that I don't know @Squirrel18

Reply:
Jajaja ok.